<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProduct</class>
 <widget class="QWidget" name="AddProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
                        {
                        border-width: 2px;
                        border-radius: 5px;
                        }
                        QLineEdit:Focus {
                        border:1px solid #6443DA
                        }
                    </string>
   </property>
   <property name="placeholderText">
    <string>   Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>461</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>36</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Добавление товара</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAmount">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
                        {
                        border-width: 2px;
                        border-radius: 5px;
                        }
                        QLineEdit:Focus {
                        border:1px solid #6443DA
                        }
                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>   Количество </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDescription">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
                        {
                        border-width: 2px;
                        border-radius: 5px;
                        }
                        QLineEdit:Focus {
                        border:1px solid #6443DA
                        }
                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>   Описание (до 100 символов)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCategory">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
                        {
                        border-width: 2px;
                        border-radius: 5px;
                        }
                        QLineEdit:Focus {
                        border:1px solid #6443DA
                        }
                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>   Категория </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrice">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
                        {
                        border-width: 2px;
                        border-radius: 5px;
                        }
                        QLineEdit:Focus {
                        border:1px solid #6443DA
                        }
                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>   Цена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addProductButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    QPushButton {
        background-color: #4BB34B;
        color: white;
        border-radius: 8px;
        padding: 10px 20px;
    }
    QPushButton:hover {
        background-color: #5fd35f;
    }
    QPushButton:pressed {
        background-color: #3fa53f;
    }</string>
   </property>
   <property name="text">
    <string>Добавить товар</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    QPushButton {
        background-color: transparent;
        color: white;
        border: 1px solid #2e2e2e;
        border-radius: 8px;
        padding: 10px 20px;
    }
    QPushButton:hover {
        border-color: #4e4e4e;
    }
    QPushButton:pressed {
        background-color: #1e1e1e;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="productLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { color: gray; }</string>
   </property>
   <property name="text">
    <string>Продукт успешно добавлен</string>
   </property>
  </widget>
  <zorder>lineEditCategory</zorder>
  <zorder>lineEditName</zorder>
  <zorder>label</zorder>
  <zorder>lineEditAmount</zorder>
  <zorder>lineEditDescription</zorder>
  <zorder>lineEditPrice</zorder>
  <zorder>addProductButton</zorder>
  <zorder>backButton</zorder>
  <zorder>productLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
